docs/concepts/PSRule/en-US/about_PSRule_Options.md

```diff
@@ -916,6 +917,52 @@ variables:
   value: false
 ```
 
+### Execution.InvariantCultureWarning
+
+When evaluating rules inside a Linux CI host, if invariant culture is used, a warning is shown by default.
```

Comment:
I did notice that this warning can also show up in Windows CI, so maybe I should omit `Linux` here?

Reply:
Good idea, let's not specifically mention Linux.